### ISSUE TITLE
general question design

### ISSUE THREAD

**Issue by StevenClontz** [TeamBasedInquiryLearning/checkit-tbil-la#39]
I'm having some trouble communicating when aspects of a problem that we prompted for are missing, e.g. "write a statement equivalent...". Rewriting the templates so each prompt is a letter would help, and if the problem has multiple "subversions", each could be numbered. E.g.

```
1. a. Write a sentence equiv to...
1. b. Find this solution set...
2. a. Write a sentence equiv to...
2. b. Find this solution set...
```

**Comment by siwelwerd** [comment]
Generally, the prompts are much more complicated than before, and I am inclined to simplify many of the prompts which would also avoid this problem.  

For instance, the other questions use (a), (b), and (c) to distinguish questions being asked, while LE3 for some reason labels systems with a. b. and c.  and then has unlabeled bullets for questions.

**Comment by StevenClontz** [comment]
Yeah, and this issue is exacerbated by a really bad implementation of `<ol/>` in the current CheckIt (forces letters, even if nested). https://github.com/StevenClontz/checkit/issues/54
